site/_includes/layouts/base.njk

```diff
@@ -9,7 +9,8 @@
 
     {% include 'partials/meta.njk' %}
     {% block feed %}
-      <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="{{ site.url }}/feeds/blog.xml">
+      <link rel="alternate" type="application/rss+xml" title="{{ site.title }} - {{ 'i18n.common.blog' | i18n(locale) }}" href="{{ site.url }}/feeds/blog.xml">
```

Comment:
Since this is the main layout, shouldn't we link to all.xml instead?
@rachelandrew wdyt?

Reply:
Checking the contents of that all feed, it seems utterly broken. This might explain why it wasn’t exposed in the first place.

Reply:
I think many RSS readers will just pick the first linked feed anyways, and ignore the other though. So it would be better to fix all.xml, or combine blog + articles in one feed instead of exposing both urls.

Reply:
Good call. I can easily correct the “all” feed generation and list it first.